lab01.py

```diff
@@ -1 +1,4 @@
-print('Hello world')
\ No newline at end of file
+integer_1 = int(input("Enter one integer: "))
+integer_2 = int(input("Enter one integer: "))
+integer_3 = int(input("Enter one more integer: "))
+print("You put in: ", integer_1, integer_2, integer_3)
```

Comment:
Looking good! A few nitpicks:

When I run the program, the output looks like this:
```python
➜  python lab01.py
Enter one integer: 1
Enter one integer: 2
Enter one more integer: 3
('You put in: ', 1, 2, 3)
```
It looks kind of weird with the final message being in parentheses, with all the punctuation from the code. Is there a way to print the message without all that?

Reply:
Nevermind, that's just a python2/3 difference!

How about this:

`You put in:  1 2 3`

There are two spaces here after the colon - can we make it so there's just one space?